docs/index.rst

```diff
@@ -25,20 +25,21 @@ you enjoy developing for the BBC micro:bit using MicroPython.
     **V2**.
 
 If you're a new programmer, teacher or unsure where to start, begin with the
-tutorials.
-
-.. image:: comic.png
+:ref:`Tutorials <Tutorials>` and use the `micro:bit Python Editor <https://python.microbit.org>`_
+to program the micro:bit. 
 
 .. note::
 
-    This project is under active development. Please help other
-    developers by adding tips, how-tos, and Q&A to this document.
-    Thanks!
+   MicroPython will not work in the MakeCode editor, as this
```

Comment:
Should we say the "MicroPython API will" to clarify it's the same language, but the modules/API is different?